### PR TITLE
Adding new pages titles #181, preview for candidate ERC #151 and disable synchronized scrolling #183

### LINKED ISSUE #151: Add "Preview" link in the creation view
When looking at the current candidate draft, e.g. https://o2r.uni-muenster.de/#/createERC/lnXZx, then I must manually change the URL to get a preview, i.e. https://o2r.uni-muenster.de/#/erc/lnXZx

This way it is hard for authors to test-run their ERC without publishing it first.

Related to #149

### LINKED ISSUE #181: Updtate title to match the current view
The HTML title shown on the browser tabs currently is always "o2r", so it is impossible to tell which tab shows which ERC, for example.

image

Better tab names would be something like "Great data analysis workflow in ERC ABC12 | o2r Demo Server" and "Create ERC | o2r Demo Server", with the slug after the `|` being configurable.

### LINKED ISSUE #183: Allow disabling of synchronised scrolling in the result view
Sometimes, the documents differ to much for the synched scrolling to work. Then the reader should have the option to disable the synching so she can adjust manually.

Example: https://o2r.uni-muenster.de/#/erc/GWME2voTDb5oeaQFuTWMCEMveKS1MiXm

image

### PR DESCRIPTION
**Pull request by Fmazin**
This PR should resolve #181 and close #151. It will also resolve #183.

### CODE REVIEW

**Comment by nuest** [comment]
@Fmazin if you want to close multiple issues, you must add a "closing keyword" in front of each issue number: 

image

File: ui/src/components/app/footerLinks/Impressum.js
Comment (nuest) [inline ui/src/components/app/footerLinks/Impressum.js:5]: Not sure if this is common within the UI (@NJaku01) but I'd prefer the simpler `componentMounted()` over using "did".

File: ui/src/components/app/footerLinks/Impressum.js
Comment (nuest) [inline ui/src/components/app/footerLinks/Impressum.js:6]: This applies here and multiple other places: We want `"| o2r Demoserver"` to be defined only _once_. Imagine you installing this on your own server and you want to change the name, this requires you to go into multiple files, which makes it harder to keep your codebase up to date etc.

This should go into the configuration file at https://github.com/o2r-project/o2r-UI/blob/master/ui/src/helpers/config.json

File: ui/src/components/erc/erc.css
Comment (nuest) [inline ui/src/components/erc/erc.css:7]: We've created a little color palette way back, it might be worth to take an orange from it that works nicely with our "blue". See #192

File: ui/src/components/startpage/Startpage.js
Comment (nuest) [inline ui/src/components/startpage/Startpage.js:14]: This demonstrates why it is important to change the same string in one location only: "Demo Server" vs. "Demoserver" is inconsistent.

**Review by nuest — CHANGES_REQUESTED** [review]

**Review by NJaku01 — CHANGES_REQUESTED** [review]
@Fmazin  Overall, I like the changes and how they are implemented. Thank you. I agree with the most hints from Daniel, could u revise them. I found another improvement, which I would like to talk with you. Plese contact me over Mattermost.

File: ui/src/components/erc/Check/Comparison/Comparison.js
Comment (nuest) [reply ui/src/components/erc/Check/Comparison/Comparison.js:72]: @NJaku01 I was not aware that `componentDidMount` is an external function, I wrongly assumed that you named that function yourself. Sorry, keep as it is, please.

File: ui/src/components/app/footerLinks/Impressum.js
Comment (nuest) [reply ui/src/components/app/footerLinks/Impressum.js:5]: I didn't know this function was part of ReactJS - ignore my comment: https://reactjs.org/docs/react-component.html#componentdidmount

File: ui/src/components/app/footerLinks/Impressum.js
Comment (Fmazin) [reply ui/src/components/app/footerLinks/Impressum.js]: The String " | o2r Demoserver" is now defined in the config JSON (ui/src/helpers/config.json) and is used for the document titles.

https://github.com/Fmazin/o2r-UI/blob/03680d383ffa68ed31cb9c8d7c548982ba31dc85/ui/src/helpers/config.json#L4

It is used in this way: 
```
    componentDidMount() {
        this.getCompendia();
        document.title = "Author View" + config.title;
    }
```

File: ui/src/components/erc/erc.css
Comment (Fmazin) [reply ui/src/components/erc/erc.css]: I have used the color[ #CE5100 ](https://www.color-hex.com/color/ce5100 )which is defined in the the palette for warnings.

**Comment by Fmazin** [comment]
I have updated the look of the preview notice a little bit. 
Bildschirmfoto von 2020-12-21 16-09-03

File: ui/src/components/erc/erc.css
Comment (nuest) [reply ui/src/components/erc/erc.css]: Good!

**Review by nuest — APPROVED** [review]
LGTM! Please remove the "Draft" when you think you're done.

Can you please confirm that you included the changes you and @NJaku01 wanted to discuss in the chat? Thakns!

File: ui/src/components/erc/erc.css
Comment (NJaku01) [inline ui/src/components/erc/erc.css:7]: You added the color here again as css. It is also possible to use the color, specified in the theme.js. This would be the better way. Because If we want to change the color, we just have to change the color at one place and not in every css file. I just have inspected the UI and saw that there is also some code implemented this way. I have created a new issues #197, maybe we can postpone this and check, update the whole UI afterwards.

**Review by NJaku01 — CHANGES_REQUESTED** [review]

**Comment by NJaku01** [comment]
The changes from the chat are included. Thanks!

During the review, I recognized two further possible improvements. The first would good to change before the merge. The other one is a general propblem within the UI. I have created a further issue #197 for this.

File: ui/src/components/createERC/CreateERC.js
Comment (Fmazin) [reply ui/src/components/createERC/CreateERC.js]: I have removed the goToPreview Function and replaced it with goToERC

**Review by NJaku01 — APPROVED** [review]
Everything looks good now

**Pull request merged**
